examples/plot_bcic_iv_2a_moabb_cropped.py

```diff
@@ -0,0 +1,135 @@
+"""
+Cropped Decoding on BCIC IV 2a Competition Set with skorch and moabb.
+=====================================================================
+"""
+
+# Authors: Maciej Sliwowski <maciek.sliwowski@gmail.com>
+#          Robin Tibor Schirrmeister <robintibor@gmail.com>
+#          Lukas Gemein <l.gemein@gmail.com>
+#          Hubert Banville <hubert.jbanville@gmail.com>
+#
+# License: BSD-3
+from collections import OrderedDict
+from functools import partial
+
+import numpy as np
+import torch
+import mne
+from skorch.callbacks import LRScheduler
+mne.set_log_level('ERROR')
+
+from braindecode.datautil.windowers import create_windows_from_events
```

Comment:
```suggestion
from braindecode.datautil import create_windows_from_events
```
let's avoid 3 layers of nesting for import of this one